src/connection/sqlJob.ts

```diff
@@ -209,6 +221,27 @@ export class SQLJob {
     return version;
   }
 
+  async explain(statement: string, type: ExplainType = ExplainType.Run): Promise<ExplainResults<any>> {
+    const explainRequest = {
+      id: SQLJob.getNewUniqueId(),
+      type: `dove`,
+      sql: statement,
+      run: type === ExplainType.Run
+    }
+
+    const result = await this.send(JSON.stringify(explainRequest));
+
+    const explainResult: ExplainResults<any> = JSON.parse(result);
+
+    if (explainResult.success !== true) {
+      this.dispose();
```

Comment:
This line and the next should be removed, as it unnecessarily closes the channel